docs/examples/error_snapshotter.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: error-snapshotter
+title: Error snapshotter
```

Comment:
Could you please come up with a better name? Perhaps "Capturing error snapshots in crawlers"? Refer to the titles of other example pages for inspiration (the crawler-related titles might be exceptions).

Why - If I do not know what "Error snapshotter" is, I don't know I need it. This is a title more for an API docs page.